consul/base.py

```diff
@@ -640,13 +646,38 @@ def delete(self, key, recurse=None, cas=None, token=None, dc=None):
             return self.agent.http.delete(
                 CB.json(), '/v1/kv/%s' % key, params=params)
 
+    class TXN(object):
```

Comment:
Do you mean this: `class Transaction(object)` for definition. But in the client itself it's `consul.txn`?